src/Selenium2Driver.php

```diff
@@ -802,6 +802,19 @@ public function isVisible($xpath)
             ($this->elementIsFullyInViewport($element) || $this->elementIsPartiallyInViewport($element));
     }
 
+    /**
+     * Checks whether element is displayed located by it's XPath query.
+     *
+     * @param string $xpath
+     * @return Boolean
```

Comment:
PHP scalar types are always lowercase. Also, our standard is to use `bool` instead of the full word.

- [ ] Done

Reply:
Done.